lib/datasource/rubygems/index.spec.ts

```diff
@@ -46,6 +52,23 @@ describe(getName(), () => {
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
 
+    it('returns null for 404', async () => {
+      httpMock
+        .scope('https://firstparty.com')
+        .get('/basepath/api/v1/gems/rails.json')
+        .reply(404, null)
+        .get('/basepath/api/v1/dependencies?gems=rails')
+        .reply(404, null);
+      httpMock
+        .scope('https://thirdparty.com')
+        .get('/api/v1/gems/rails.json')
+        .reply(404, null)
+        .get('/api/v1/dependencies?gems=rails')
+        .reply(404, null);
+      expect(await getPkgReleases(params)).toBeNull();
+      expect(httpMock.getTrace()).toMatchSnapshot();
```

Comment:
```suggestion
```

no longer required, can be removed from all tests (in a later pr)

Reply:
Done in eb9b852